packages/module/src/FileDetailsLabel/FileDetailsLabel.scss

```diff
@@ -27,8 +27,8 @@
 
 .pf-chatbot__file-label.pf-m-clickable:hover,
 .pf-chatbot__file-label.pf-m-clickable:focus-within {
-  --pf-v6-c-label--BackgroundColor: var(--pf-t--global--background--color--action--plain--hover);
-  --pf-v6-c-label--m-clickable--hover--BackgroundColor: var(--pf-t--global--background--color--action--plain--hover);
+  --pf-v6-c-label--BackgroundColor: var(--pf-t--global--background--color--tertiary--default);
```

Comment:
Hey! If I am reading this correctly, I think this is the hover token for the attachment label right? We can actually keep the --pf-t--global--background--color--action--plain--hover applied since it's semantically correct to the hover state. The tertiary--default is the exact same value at the action--plain but the semantics and usage guidelines slightly differ!

Reply:
All set!